### DISCUSSION TITLE
sykz's submission(chapter3)

### DISCUSSION THREAD

**Discussion by Sykzpixel** [stellar/sorobanathon#96] [General]
https://litemint.com/items/GDPLA6GS6UPCPNF2TKNJLZNK4LF5NFI5YPYS4AYEJXVCLFH3JADKMAYI/sykz2

**Comment by ElliotFriend** [comment]
These are amazing!!! Both of your chapter1 and chapter3 images are **incredible**!! Just wow!

**Reply by Sykzpixel** [reply]
Thank youu

**Comment by kalepail** [comment]
The little conversation type outs are an incredible touch 👏

**Reply by Sykzpixel** [reply]
Thank you ^^

**Comment by stellarjohn** [comment]
Hi Sykz,

Congratulations! Your contribution to has been accepted! 🎉
Your submission may qualify for a reward of 300 XLM! One last step: to receive that reward, please fill out this [form](https://stellarform.typeform.com/Sorobanathon) completely and accurately.

Please let us know if you have any questions.

Cheers!
John
